doc/tutorials/add_solver.rst

```diff
@@ -234,24 +234,21 @@ Specifying the solver parameters
 If your solver has hyperparameters, you can specify them by adding an attribute ``parameters``.
 This attribute is a dictionary whose keys are the solver's hyperparameters.
 
-For example, if our solver has two hyperparameters, ``stepsize`` and ``momentum``, we implement them as follows:
```

Comment:
this is just an illustration, doesn't have to match the solver we implemented. It comes at the end of the tuto so people should be more familiar with the principle.

The proposed change is not great either because `positive` changes the optimization problem/